templates/prelogin/_sections/software_services/shared_pricing_table.html

```diff
@@ -0,0 +1,162 @@
+{# This file alone is available under the BSD 3-Clause license as posted at #}
+{# https://github.com/dimagi/commcare-hq/blob/master/LICENSE #}
```

Comment:
My gut says it's fine, but I'm not a lawyer, so I of course can't say definitively.